app/controllers/LibFriends.js

```diff
@@ -74,7 +74,6 @@ function renderFriendsFeed(options, callback) {
   const params = {
     after: options.after,
     before: options.before,
-    //limit:limit
```

Comment:
I don't remember why I has left that commented code (the joys of legacy code...), but it was probably for some reason => can you cancel this change, please?